README.md

```diff
@@ -74,3 +74,15 @@ run this code in external terminal to run tests
  poetry run
 pytest path/to/test_file
 ```
+## Docker
+To build both production and development run
+```
+$ docker build --target development --tag todo-app:dev .
+$ docker build --target production --tag todo-app:prod .
+```
+and these commands to run it
+```
+docker run --env-file ./.env -p 5100:80 --mount type=bind,source="$(pwd)"\todo_app,target=/todo_app/app.py todo-app:dev
+
+docker run --env-file ./.env -p 5100:80 --mount type=bind,source="$(pwd)"\todo_app,target=/todo_app/app.py todo-app:prod
```

Comment:
Have you tried running these since adjusting the commands? The target still doesn't line up quite right for me; if you look at the structure of your containers (e.g. running `docker run -it --entrypoint bash todo-app:dev` and then exploring) then the files are like:
/todo_app
  - pyproject.toml
  - poetry.lock
  - ...
  - /todo_app
    - app.py
    - ...
So I think you want to fix the dev one like below, and potentially just drop the bind mount for production since we expect the code to be copied in for the prod build and it shouldn't be being played with on the fly:
```suggestion
docker run --env-file ./.env -p 5100:80 --mount type=bind,source="$(pwd)"\todo_app,target=/todo_app/todo_app todo-app:dev

docker run --env-file ./.env -p 5100:80  todo-app:prod
```